src/community/geofence-server/src/main/resources/applicationContext.xml

```diff
@@ -155,32 +155,10 @@
     <mvc:interceptors>
        <bean class="org.geoserver.geofence.rest.GeofenceSecurityInterceptor"/>
     </mvc:interceptors>
-  
-    <bean id="geofenceViewResolver" class="org.springframework.web.servlet.view.ContentNegotiatingViewResolver">
-        <property name="order" value="1" />
-        <property name="defaultViews">
-            <list>
-                <!-- XML view using a JAXB marshaller -->
-                <bean id="jaxbView" class="org.springframework.web.servlet.view.xml.MarshallingView">
-                    <constructor-arg>
-                        <bean id="jaxb2Marshaller" class="org.springframework.oxm.jaxb.Jaxb2Marshaller">
-                            <property name="classesToBeBound">
-                                <list>
-                                    <value>org.geoserver.geofence.rest.xml.JaxbRule</value>
-                                    <value>org.geoserver.geofence.rest.xml.JaxbRuleList</value>
+
                                     <value>org.geoserver.geofence.rest.xml.JaxbRoleList</value>
```

Comment:
@NielsCharlier one other question (after I merged the PR!): these lines look out of place and are likely ignored. You have removed the enclosing bean.

Reply:
You're right, that is odd. Bad merge. Fixing.